neo-gui/GUI/MainForm.cs

```diff
@@ -0,0 +1,578 @@
+            {
+                if (dialog.ShowDialog() != DialogResult.OK) return;
+                tx = dialog.GetTransaction();
+            }
+            using (InvokeContractDialog dialog = new InvokeContractDialog(tx))
+            {
+                if (dialog.ShowDialog() != DialogResult.OK) return;
+                tx = dialog.GetTransaction();
+            }
+            Helper.SignAndShowInformation(tx);
+        }
+
+        private void 签名SToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            using SigningTxDialog dialog = new SigningTxDialog();
+            dialog.ShowDialog();
+        }
+
+        private void deployContractToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                byte[] script;
+                using (DeployContractDialog dialog = new DeployContractDialog())
+                {
+                    if (dialog.ShowDialog() != DialogResult.OK) return;
+                    script = dialog.GetScript();
+                }
+                using (InvokeContractDialog dialog = new InvokeContractDialog(script))
+                {
+                    if (dialog.ShowDialog() != DialogResult.OK) return;
+                    Helper.SignAndShowInformation(dialog.GetTransaction());
+                }
+            }
+            catch { }
+        }
+
+        private void invokeContractToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            using InvokeContractDialog dialog = new InvokeContractDialog();
+            if (dialog.ShowDialog() != DialogResult.OK) return;
+            try
+            {
+                Helper.SignAndShowInformation(dialog.GetTransaction());
+            }
+            catch
+            {
+                return;
+            }
+        }
+
+        private void 选举EToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                byte[] script;
+                using (ElectionDialog dialog = new ElectionDialog())
+                {
+                    if (dialog.ShowDialog() != DialogResult.OK) return;
+                    script = dialog.GetScript();
+                }
+                using (InvokeContractDialog dialog = new InvokeContractDialog(script))
+                {
+                    if (dialog.ShowDialog() != DialogResult.OK) return;
+                    Helper.SignAndShowInformation(dialog.GetTransaction());
+                }
+            }
+            catch { }
+        }
+
+        private void signDataToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            using SigningDialog dialog = new SigningDialog();
+            dialog.ShowDialog();
+        }
+
+        private void optionsToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+        }
+
+        private void 官网WToolStripMenuItem_Click(object sender, EventArgs e)
+        {
+            Process.Start("https://neo.org/");
```

Comment:
I can't open the browser in my Windows
![image](https://user-images.githubusercontent.com/3167973/70052802-6762ce00-15d4-11ea-8480-7cc897cc3703.png)

I found this issue https://github.com/dotnet/corefx/issues/10361

```suggestion
            OpenBrowser("https://neo.org/");
        }

        public static void OpenBrowser(string url)
        {
            if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
            {
                Process.Start(new ProcessStartInfo("cmd", $"/c start {url}"));
            }
            else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux))
            {
                Process.Start("xdg-open", url);
            }
            else if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
            {
                Process.Start("open", url);
            }
        }
        
```